app.rb

```diff
@@ -0,0 +1,97 @@
+require_relative './lib/book'
+require_relative './lib/student'
+require_relative './lib/teacher'
+require_relative './lib/rental'
+
+class Option
+  def initialize
+    @books = []
+    @person = []
+    @rents = []
+  end
+
+  def list_books
+    puts "There is no book yet! Please create a new book first. \n" if @books.empty?
+
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    puts "There is no person yet! Please create a new person first \n" if @person.empty?
+    @person.each do |each|
+      puts " [#{each.class}] Name: #{each.name}, ID: #{each.id}, Age: #{each.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student(1) or a teacher(2)? [Input the number]: '
+    type = gets.chomp.to_i
+
```

Comment:
- [ ] [OPTIONlAL]
Let's improve upon the interactivity of this app by ensuring that user inputs are placed on a new line.

![image](https://user-images.githubusercontent.com/12745474/233567393-dde4fc33-2477-43b9-ad69-d74d9f3011f3.png)